src/helpers/config/pageTitles.js

```diff
@@ -0,0 +1,9 @@
+const titles = {
+  '/latest-rules/': 'Latest Rules',
+  '/user/': 'User Rules',
+  '/orphaned/': 'Orphaned Rules',
+  '/archived/': 'Archived Rules',
+  '/profile/': 'Profile',
+};
+
+export default titles;
```

Comment:
Should we move this to https://github.com/SSWConsulting/SSW.Rules/blob/main/site-config.js

Just so its all in one spot?